src/theory/quantifiers/sygus/sygus_enumerator.cpp

```diff
@@ -54,6 +57,12 @@ void SygusEnumerator::initialize(Node e)
   d_tlEnum = getMasterEnumForType(d_etype);
   d_abortSize = options::sygusAbortSize();
 
+  // if we don't have a term database, we don't register symmetry breaking
+  // lemmas
+  if (d_tds == nullptr)
```

Comment:
```suggestion
  if (!d_tds)
```
This style could be used for the numerous similar tests below as well.